core/new-gui/src/app/dashboard/user/component/user-project/user-project-section/user-project-section.component.ts

```diff
@@ -202,12 +211,15 @@ export class UserProjectSectionComponent implements OnInit {
    * @param userFileEntry
    */
   public deleteUserFileEntry(userFileEntry: DashboardFile): void {
+    if (!isDefined(this.pid)) {
+      return;
+    }
     this.userProjectService.deleteDashboardUserFileEntry(this.pid, userFileEntry);
   }
 
   private getUserProjectMetadata() {
     this.userProjectService
-      .retrieveProject(this.pid)
+      .retrieveProject(this.pid!)
```

Comment:
still got one case of `!` (non-null assertion).

Reply:
fixed